[2025-02-06T15:37:47.411+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-06T15:37:47.424+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_que_faire_a_paris.extract_data manual__2025-02-06T15:37:46.079732+00:00 [queued]>
[2025-02-06T15:37:47.429+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_que_faire_a_paris.extract_data manual__2025-02-06T15:37:46.079732+00:00 [queued]>
[2025-02-06T15:37:47.432+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-06T15:37:47.442+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_data> on 2025-02-06 15:37:46.079732+00:00
[2025-02-06T15:37:47.447+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=8066) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-06T15:37:47.449+0000] {standard_task_runner.py:72} INFO - Started process 8068 to run task
[2025-02-06T15:37:47.448+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_que_faire_a_paris', 'extract_data', 'manual__2025-02-06T15:37:46.079732+00:00', '--job-id', '111', '--raw', '--subdir', 'DAGS_FOLDER/dag_que_faire_a_paris.py', '--cfg-path', '/tmp/tmph3bwvet7']
[2025-02-06T15:37:47.450+0000] {standard_task_runner.py:105} INFO - Job 111: Subtask extract_data
[2025-02-06T15:37:47.476+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_que_faire_a_paris.extract_data manual__2025-02-06T15:37:46.079732+00:00 [running]> on host b176106a6d6f
[2025-02-06T15:37:47.616+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_que_faire_a_paris' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-06T15:37:46.079732+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-06T15:37:46.079732+00:00'
[2025-02-06T15:37:47.619+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-06T15:37:48.116+0000] {dag_que_faire_a_paris.py:40} INFO - âœ… Extraction completed successfully!
[2025-02-06T15:37:48.118+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-06T15:37:48.124+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-06T15:37:48.124+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_que_faire_a_paris, task_id=extract_data, run_id=manual__2025-02-06T15:37:46.079732+00:00, execution_date=20250206T153746, start_date=20250206T153747, end_date=20250206T153748
[2025-02-06T15:37:48.145+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-06T15:37:48.176+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-06T15:37:48.184+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
